jwst/datamodels/schemas/multispec.schema.yaml

```diff
@@ -21,7 +21,7 @@ allOf:
             - name: ERROR
               datatype: float64
             - name: DQ
-              datatype: int32
+              datatype: uint32
```

Comment:
I wonder if this will break any existing truth files we have in the reg tests. Data models might throw a validation error now when it tries to load them. Oh well, it has to be done.

Reply:
Yes, but so will adding the NPIXELS column.  That's why I delayed making the change to DQ until also adding NPIXELS, i.e. break them only once.

Reply:
Good point. Hadn't even thought about the extra column.